dotCMS/src/integration-test/java/com/dotcms/content/business/ObjectMapperTest.java

```diff
@@ -0,0 +1,153 @@
+package com.dotcms.content.business;
+
+import com.dotcms.util.IntegrationTestInitService;
+import com.fasterxml.jackson.annotation.JsonView;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+/**
+ * Here a small test created for the purpose of experimenting wight jackson mapper and it's capabilities
+ */
+public class ObjectMapperTest {
```

Comment:
This new test needs to be added to the MainSuite in order to run.

Reply:
@erickgonzalez done. Please continue with your review